### DISCUSSION TITLE
Merge volumes

### DISCUSSION THREAD

**Discussion by hamidreza-hosseini** [seaweedfs/seaweedfs#2647] [Q&A] [answered]
I grow my volumes from a specific collection to a huge number, now all volumes have a little data and I couldn't delete them
and they are filling and  occupying my volumes max number,
Is it possible to merge volumes with the same collection that are on the same disk?

**Comment by chrislusf** [comment, marked as answer]
not possible, unless there is a way to go through all files and change all affected file id to switch the volume id.
